deploy/catalog/README.md

```diff
@@ -4,31 +4,77 @@ The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
 
 ## Prerequisites
 
-  - [Helm](https://github.com/kubernetes/helm) must be installed in the cluster.
-  - Images must be built from source and pushed to a registry accessible to
-    the cluster. Example (from this directory):
-
-    ```
-    $ export REGISTRY=hub.docker.com/<username>
-    $ make -C ../.. push
-    ```
-
-    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
-    to use a _local_ Docker registry.
+- [Helm](https://github.com/kubernetes/helm) must be installed in the cluster
+- Images must be built from source and pushed to a registry accessible to the
+  cluster if deploying to a non-local cluster; see the [dev guide](../../docs/DEVGUIDE.md) for instructions
 
 ## Usage
 
 Supported template parameters (values):
 
-  - `registry`  (required): Container registry with Service Catalog images.
-  - `version`   (optional): Version of Service Catalog (container images) to deploy.
-  - `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment.
-  - `storageType` (optional): The type of storage for the API server to use.
-  - `verbosity` (optional): The verbosity of logs.
-  - `debug` (optional): create a load balancer for the api server service
+### Deployment Knobs
+
+- `registry`: Container registry with Service Catalog images; optional; defaults
+  to unset
+- `namespace`: A Kubernetes namespace to use for deployment of this
+  chart into k8s; optional; defaults to `default`
+- `version`: Version of Service Catalog (container images) to deploy; optional;
+  defaults to `latest`
+- `apiServerVersion`: The version of the API server image to deploy; optional;
```

Comment:
So... this isn't obvious to everyone right now because we're overusing command line flags to set options, but the way options are generally set is by passing in a `values.yaml` that has been copied from the chart and modified. Teams tend to check these `values.yaml` (which may contain sensitive details) into private git repos so that they have a re-replayable record of the arguments that were used with their charts when they were installed. This comes in handy later when, for instance, wishing to upgrade and re-use all the same options as previously.

Back to my point...

Since the options are really YAML, there's the option to structure them.

I would recommend that instead of having an option like `apiServerVersion`, we instead have `version` under `apiServer`. We should aggregate _other_ options that affect the apiserver under `apiServer` as well. Same goes for controller. And etcd. On the command line, for those who will continue to use the `--set` flag, these options can be references, for instance, as `apiServer.version`